frontend/src/Dashboard.tsx

```diff
@@ -112,13 +111,12 @@ const Dashboard = () => {
             const jobDetailsResponse = await axios.get(API_ROUTES.JOBS, {
               params: { id: myApplications.job },
             });
-            const jobDetail = jobDetailsResponse.data;
-
-            const locationDetailsResponse = await axios.get(
-              `${API_ROUTES.USER.LOCATION}?location_id=${jobDetail.location}`
-            );
```

Comment:
We need to send location id in the request to get the correct response.

Reply:
ya i am reverting it to your changes. Somehow when i pulled and pushed again your changes  were overwritten by mine.